### ISSUE TITLE
[suggestion] Redundant Item Details Header

### ISSUE THREAD

**Issue by neopc10** [jarnedemeulemeester/findroid#20]
![image](https://user-images.githubusercontent.com/1365322/130370030-5bfd1841-ca4a-4fd8-938f-5ec836913bf6.png)

The header on top would benefit from two tweaks:
1. Remove the item name, as that info is already provided elegantly just below.
2. Make the header bar either mostly or completely transparent so that the backdrop does not get cut off. In the above example, it's glaring.

I don't think the "home" and "my media" headers are necessary either, but that's minor.

**Comment by jarnedemeulemeester** [comment]
Yes indeed I agree. In the original design there was no top bar. 
The reason there is a top bar is because I'm using one top bar for all screens. 

I agree it's redundant and could be improved!

**Comment by jarnedemeulemeester** [comment]
Resolved in the compose rework
